virttest/qemu_devices/qdevices.py

```diff
@@ -323,8 +323,9 @@ def verify_unplug(self, out, monitor):      # pylint: disable=W0613,R0201
     def is_pcie_device(self):
         """Check is it a pcie device"""
         driver = self.get_param("driver", "")
-        if (driver in ("e1000e", "vhost-vsock-pci", "qemu-xhci", "vfio-pci")
-                or driver.startswith("virtio-")):
+        pcie_drivers = ["e1000e", "vhost-vsock-pci", "qemu-xhci", "vfio-pci",
+                        "vhost-user-fs-pci"]
+        if (driver in pcie_drivers or driver.startswith("virtio-")):
```

Comment:
LGTM.
BTW, we can improve the code:
```suggestion
        return (driver in pcie_drivers or driver.startswith("virtio-"))
```

Reply:
Update.